### DISCUSSION TITLE
Whether the TPC-DS test is supported

### DISCUSSION THREAD

**Discussion by qicaiyulin** [databendlabs/databend#6980] [Q&A] [answered]
We want to evaluate and measure the performance of databend using the TPC-DS benchmark. However we found that most SQL in the tpc-ds is not supported by databend. I want to know whether the TPC-DS test is supported?

**Comment by Xuanwo** [comment]
cc @leiysky @sundy-li for help.

**Comment by Xuanwo** [comment, marked as answer]
Hi, we don't support the TPC-DS test for now. @leiysky has created a tracking issue for it: https://github.com/datafuselabs/databend/issues/6984

Please feel free to leave comments there if you have any other questions!

Thanks for evaluating!
